opennms-base-assembly/src/main/filtered/bin/find-java.sh

```diff
@@ -1,7 +1,7 @@
-#!/bin/sh
+#!/bin/bash
 
-JAVA_SEARCH_DIRS="$JAVA_SEARCH_DIRS /usr/lib/jvm /usr/java /System/Library/Java/JavaVirtualMachines /Library/Java/JavaVirtualMachines /Library/Java/Home /opt"
-# set DEBUG=true to enable debugging
+# if $JAVA_SEARCH_DIRS is already set, make sure it is treated as an array
+declare -p JAVA_SEARCH_DIRS 2>/dev/null | grep -q '^declare \-a' || IFS=" " read -r -a JAVA_SEARCH_DIRS <<< "${JAVA_SEARCH_DIRS[@]}"
```

Comment:
Should this be updated to use the new `__onms_convert_array()`

Reply:
ah, yeah, good catch